plugins/Login/Login.php

```diff
@@ -143,6 +147,14 @@ public function onFailedAPILogin()
         }
     }
 
+    public function checkLoginFromAnotherCountry($login)
+    {
+        $loginFromDifferentCountryDetection = StaticContainer::get(LoginFromDifferentCountryDetection::class);
```

Comment:
We should skip this if `$login` is `anonymous`. Currently the system also sends emails to the anonymous user if the country changes 🙈